src/wallet/websocket.ts

```diff
@@ -0,0 +1,147 @@
+/**
+ * Copyright (c) Hathor Labs and its affiliates.
+ *
+ * This source code is licensed under the MIT license found in the
+ * LICENSE file in the root directory of this source tree.
+ */
+
+import BaseWebSocket, {
+  WsOptions,
+  DEFAULT_WS_OPTIONS,
+} from '../websocket/base';
+
+export interface WalletServiceWebSocketOptions extends WsOptions {
+  walletId: string;
+};
+
+const JOIN_TIMEOUT = 5000;
+
+/**
+ * Handles websocket connections and message transmission
```

Comment:
We should improve the description in this class and in class `WalletWebSocket`. They currently have the same description, so it's hard to know the different between them, and in which cases each one should be used.

Reply:
I agree, improved the documentation for both classes on 8e3c6f98808c1982f33ba1546e0e985e27e033b9